target/linux/kirkwood/modules.mk

```diff
@@ -0,0 +1,32 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+define KernelPackage/ata-ledtrig
+  TITLE:=LED trigger for ATA port
+  DEPENDS:=@TARGET_kirkwood
+  KCONFIG:=CONFIG_ATA_LEDS=y
```

Comment:
If this is put here with =y, how will this be different from having it set in config-5.4 directly?

Reply:
Because it is depended on ata-core and every kernel refresh will remove it from config.